R/teal_slices.R

```diff
@@ -9,21 +9,21 @@
 #'
 #' @inheritParams teal.slice::teal_slices
 #'
-#' @param module_specific optional (`logical(1)`)\cr
+#' @param module_specific optional (`logical(1)`)
 #'  - `FALSE` (default) when one filter panel applied to all modules.
 #'  All filters will be shared by all modules.
 #'  - `TRUE` when filter panel module-specific.
 #'  Modules can have different set of filters specified - see `mapping` argument.
-#' @param mapping `r lifecycle::badge("experimental")` _This is a new feature. Do kindly share your opinions.\cr_
-#'  (`named list`)\cr
-#'  Specifies which filters will be active in which modules on app start.
+#' @param mapping `r lifecycle::badge("experimental")` _This is a new feature. Do kindly share your opinions._
```

Comment:
Should we put in a hint on how to share opinions?

```suggestion
#' @param mapping `r lifecycle::badge("experimental")` _This is a new feature. Do kindly share your opinions on `teal`'s GitHub repository._
```

Reply:
Its good idea, but it's important to integrate it with https://github.com/insightsengineering/teal/ link. I was considering whether this is too obvious, as developers typically know where to look for posting bugs or feature requests. Is it truly necessary to explicitly mention sharing opinions?

Reply:
> Is it truly necessary to explicitly mention sharing opinions?

That was the requirement.

Reply:
how about this [6bf3d2e](https://github.com/insightsengineering/teal/pull/1052/commits/6bf3d2e021a804e64fdb77fe6ad9ba23f460a2ba).

if this okay then I will make same change for parameter `exclude_varnames` in teal.slice

Reply:
I'll move this comment to the main pre-release as this shouldn't block this PR

Reply:
Looks good to me @kartikeyakirar